style.css

```diff
@@ -0,0 +1,88 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:ital,wght@0,400;0,500;0,900;1,100&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  --lightdark: #3c3a39;
+  --white: #fff;
+  --deepDark: #1c1a19;
+}
+
+#header {
+  background: var(--lightdark);
+  height: 48px;
+  display: flex;
+  justify-content: space-between;
+  padding: 12px 24px 12px 24px;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 24px;
+}
+
+#logo {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 24px;
+  color: var(--white);
+}
+
+.header-btn {
+  background: none;
+  outline: none;
+  border: none;
+}
+
+#headline-section {
```

Comment:
The project requirement specifically asks for the use of Flexbox in both sections;
![image](https://user-images.githubusercontent.com/83596135/164359039-94bae03e-4691-46e5-9a21-5f52631cb1cb.png)

- [ ] Kindly, make use of Flexbox when you lay out the elements of this section too;